advanced_guides/stop_words.md

```diff
@@ -1,16 +1,20 @@
 # Stop words
 
-Stop-words words allow you to **ignore certain words in the relevance of your search**.
+Stop-words allows you to create a list of words **ignored in your search queries**.
 
-During a search, if your query contains words that you have put in stop-words, they will be ignored by the algorithm that defines the relevance of each document. This increases the speed of the search.
+During a search, the stop-words contained in your search query will be ignored by the sorting algorithm.
+
+I.e., if `the` is a stop word, and your search query contains `the`, the number of occurrences of `the` in any of your documents will have no impact on the search results.
+
+Stop-words **improves** **the speed** and **relevancy** of a search.
 
 ::: tip
-Using stop-words allows the relevance of the search to be focused on more interesting elements such as ranking rather than on the accuracy of the sequence with another one.
+Using stop-words allows the search to focus on more relevant elements rather than the number of occurrences of words that are very common in your dataset.
```

Comment:
```suggestion
Using stop-words allows the search to focus on more relevant elements rather than words that appears very frequently in your dataset.
```

Reply:
Great proposition